File: Practicas/Yahir-Hermosillo/Parcial3/Practica6/Professor.java
public class Professor extends MagicPerson {
    private String subject;

    public Professor(String name, int age, String subject) {
        super(name, age);
        this.subject = subject;
    }

    @Override
    public void castSpell() {
        System.out.println(getName() + ", profesor de " + subject + ", lanza un hechizo especial para enseñar.");
    }
}

File: Practicas/Marco-Oregel/Practica6/Professor.py
from MagicPerson import MagicPerson

class Professor(MagicPerson):
    def __init__(self, name, age,subject):
        super().__init__(name, age)
        self.subject = subject

    def castSpell(self):
      print(f"Soy el profesor {self.name} de la clase de {self.subject}, Petrificus Totalus!")